android_symbol_uploader.go

```diff
@@ -142,14 +141,17 @@ func (a *androidSymbolUploader) getBuildVersion(e *env) (int, error) {
 		return versionCode, nil
 	}
 
-	versionCode, err := a.getBuildVersionFromAPK()
-	if err != nil {
-		return 0, err
-	}
-	if versionCode == 1 {
-		fmt.Fprintln(e.Out, "Warning: build number is '1'.")
+	var (
+		versionCode int
+		err         error
+	)
+	if a.Manifest != "" {
```

Comment:
I would write this part like

if a.Manifest != "" {
  return f(a.getBuildVersionFromManifest)
}
return f(a.getBuildVersionFromAPK())